diff --git a/node_modules/cli-table3/src/utils.js b/node_modules/cli-table3/src/utils.js
index d21e251..ab2a822 100644
--- a/node_modules/cli-table3/src/utils.js
+++ b/node_modules/cli-table3/src/utils.js
@@ -1,4 +1,6 @@
-const stringWidth = require('string-width');
+// const stringWidth = require('string-width');
+const stringWidth = async () => (await import('string-width')).default;
+
 
 function codeRegex(capture) {
   return capture ? /\u001b\[((?:\d*;){0,5}\d*)m/g : /\u001b\[(?:\d*;){0,5}\d*m/g;
